protobuf/grpc_service.proto

```diff
@@ -1104,6 +1158,25 @@ message ModelStatistics
   //@@     compute).
   //@@
   repeated InferBatchStatistics batch_stats = 7;
+
+  //@@  .. cpp:var:: InferResponseStatistics response_stats (repeated)
+  //@@
+  //@@     The aggregate statistics for different responses that the model may
+  //@@     produce. The response statistics can provide the average of the time
+  //@@     it takes for the backend to generate each individual response. The
+  //@@     results in the response statistics are aggregated by the number of
+  //@@     responses that each request produces and the response index.
+  //@@     For example, each InferResponseStatistics contains the cumulative
```

Comment:
This line is not very accurate. Please generalize it in better way. And provide complete expected response_stats.
For example, take a model that only produces following number of responses: N, M. 
How would the response_stats look for this case?

Reply:
Yes. It is rewritten.